AngouriMath/Convenience/MathS.cs

```diff
@@ -715,6 +715,9 @@ public sealed record NewtonSetting
                     // Number of logarithms
                     res += TinyWeight(expr.Nodes.Count(entity => entity is Logf));
 
+                    // Number of phi functions
+                    res += HeavyWeight(expr.Nodes.Count(entity => entity is Phif));
```

Comment:
Wouldn't it work with normal Weight?

Reply:
No, not so well.  
For example consider expression `phi(2^x) / 2^x`. 
When weight of number of phi functions is normal, simplifier would rather leave it as it is, but when weight is heavy, we'll get a nice constant out of it: `1 - 1/2 = 1/2`.
I'm pretty sure that's because of simplifier's greediness, but i can't think of anything to help it.